liquid-bytestring/liquid-bytestring.cabal

```diff
@@ -27,8 +27,8 @@ library
                       Data.ByteString.Lazy
                       -- Data.ByteString.Lazy.Char8
   hs-source-dirs:     src
-  build-depends:      liquid-base          >= 0.8.10.1
-                    , bytestring           >= 0.10
+  build-depends:      liquid-base          < 5
+                    , bytestring           >= 0.10.0.0
                     , liquidhaskell        >= 0.8.10.1
```

Comment:
I would have expected `== 0.10.0.0`. This may be less important than for `base`. But there should definitely be an upper bound.

Reply:
Also, why `0.0`? It should be `>= 0.10.10.0` at least.